uc-crux-llvm/src/UCCrux/LLVM/Overrides/Check.hs

```diff
@@ -318,7 +340,7 @@ createCheckOverride ::
   AppContext ->
   ModuleContext m arch ->
   -- | Predicates checked during simulation
-  IORef (Map CheckOverrideName [(Stack sym, Seq (SomeCheckedConstraint m sym argTypes))]) ->
+  IORef [CheckedCall m sym arch argTypes] ->
```

Comment:
How should I think about this `IORef`? Does the order that entries are added matter? Who is ultimately consulting it?

Reply:
Added a comment.